### ISSUE TITLE
3rd party libs must have their dependencies checked and installed.

### ISSUE THREAD

**Issue by NullVoxPopuli** [angular/angular-cli#1168]
This is too big of a problem to do the 5 or so steps for being able to use a 3rd party lib.

I tried setting up sinon.

when I imported it in my test, the log showed the following:

```
22 06 2016 09:32:03.802:WARN [web-server]: 404: /base/dist/util
22 06 2016 09:32:03.807:WARN [web-server]: 404: /base/dist/formatio
22 06 2016 09:32:03.891:WARN [web-server]: 404: /base/dist/lolex
Missing error handler on `socket`.
TypeError: (msg || "").replace is not a function
... trace
22 06 2016 09:32:04.011:WARN [web-server]: 404: /base/dist/http
22 06 2016 09:32:04.013:WARN [web-server]: 404: /base/dist/https
22 06 2016 09:32:04.014:WARN [web-server]: 404: /base/dist/follow-redirects
22 06 2016 09:32:04.016:WARN [web-server]: 404: /base/dist/url
22 06 2016 09:32:04.017:WARN [web-server]: 404: /base/dist/zlib
22 06 2016 09:32:04.018:WARN [web-server]: 404: /base/dist/buffer
22 06 2016 09:32:04.023:WARN [web-server]: 404: /base/dist/vendor/axios/package.json.js
22 06 2016 09:32:04.026:WARN [web-server]: 404: /base/dist/util
22 06 2016 09:32:04.028:WARN [web-server]: 404: /base/dist/formatio
```

That is a ton of libs. A few of those are node-native things, so I wouldn't know what packages to install to achieve the same functionality.

```
$ ng --version
angular-cli: 1.0.0-beta.6
node: 5.10.1
os: linux x64
```

**Comment by filipesilva** [comment]
I'm hoping the upcoming webpack update will help with this, but for now I don't have a better solution... Sorry :/

**Comment by filipesilva** [comment]
Closed as issue was made obsolete by #1455. This now works correctly as you expected it.
